disco/disco.go

```diff
@@ -122,13 +122,49 @@ func parsePing(ver uint8, p []byte) (m *Ping, err error) {
 //
 // The recipient may choose to not open a path back, if it's already
 // happy with its path. But usually it will.
-type CallMeMaybe struct{}
+type CallMeMaybe struct {
+	// MyNumber is what the peer believes its endpoints are.
+	// Tailscale clients before 1.4 did not populate this
+	// so these values should merely augment whetever the control
+	// server sends. But because the client could've been idle
+	// before it reached out to us, the control plane might
+	// have stale info and these endpoints in CallMeMaybe
+	// might contain the just-obtained-milliseconds-ago
+	// STUN endpoint.
```

Comment:
I think this comment should the relationship a bit clearer. (It currently explains why, but not what.) IIUC, it is:

> Prefer `MyNumber` to control's data, but use both.

Reply:
included in next change